modules/nodes-secondary-scheduler-uninstall-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-uninstalling.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-uninstall-console_{context}"]
+= Uninstalling the {secondary-scheduler-operator}
+
+You can uninstall the {secondary-scheduler-operator-full} by using the web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Uninstall the {secondary-scheduler-operator-full} Operator.
+.. Navigate to *Operators* -> *Installed Operators*.
+.. Click the Options menu {kebab} next to the *{secondary-scheduler-operator}* entry and click *Uninstall Operator*.
+.. In the confirmation dialog, click *Uninstall*.
```

Comment:
@ingvagabund should we ask to delete the secondary scheduler CR before removing the operator ?

Reply:
ACK !!!